keyboards/keebio/iris/keymaps/ddone/keymap.c

```diff
@@ -0,0 +1,145 @@
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
We highly recommend using enum's over defines, as it's easier to maintain long term.  It starts at 0 and auto-increments. 
```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```